app/(admin)/layout.tsx

```diff
@@ -31,10 +35,28 @@ const RootLayout = ({
 }): JSX.Element => {
   return (
     <html lang="en" className={GeistSans.className}>
-      <body className="dark:dark bg-background pb-8 px-4 text-foreground xl:pb-0">
+      <body className="dark:dark bg-background text-foreground">
         <ErrorBoundary>
           <AuthContextProvider>
-            <main>{children}</main>
+            <main className="grid grid-cols-adminLayout h-screen">
+              <section className="left-column grid grid-rows-adminLayout border-r-2 border-border">
+                <div className="flex items-center border-b-2 border-border">
+                  <AdminLogo />
+                </div>
+                <div>
+                  <AdminNav />
+                </div>
+              </section>
+              <section className="right-column grid grid-rows-adminLayout">
+                <div className="flex items-center border-b-2 border-border px-6">
```

Comment:
This div should be inside AdminQuickMenu component.

Reply:
Added to the component